frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -476,7 +472,9 @@ void ProgramStructure::createParser() {
     conversionContext.clear();
 }
 
-void ProgramStructure::include(cstring filename) {
+void ProgramStructure::include(cstring filename, cstring ppoptions) {
+    if (included_files.count(filename)) return;
```

Comment:
Could this break the semantics of `#include` by preventing the inclusion of a file multiple times?